.circleci/config.yml

```diff
@@ -58,9 +58,6 @@ jobs:
     steps:
       - checkout
       - restore_cache: *restore-npm-cache
-      - run:
```

Comment:
always thought it was good practice to have our packages hooked on a single version until we want to change it manually.

Reply:
I can see how that can be beneficial. However, that's what we get with semantic versioning and lockfiles.

Sometimes we should avoid using `^x.x.x` and use `~x.x.x` so that we narrow down the scope.

The drawback of fixing versions is that we don't get patch updates which are often important for security fixes.